*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:13:43 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x8c9060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x8c9060 pid: 6485 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6485)
spawned uWSGI worker 1 (pid: 6489, cores: 1)
spawned uWSGI worker 2 (pid: 6490, cores: 1)
spawned uWSGI worker 3 (pid: 6491, cores: 1)
spawned uWSGI worker 4 (pid: 6492, cores: 1)
spawned uWSGI worker 5 (pid: 6493, cores: 1)
[pid: 6493|app: 0|req: 1/1] 70.213.12.152 () {42 vars in 807 bytes} [Wed Jun 29 20:13:52 2016] GET / => generated 3164 bytes in 23 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6489|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:13:52 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6489|app: 0|req: 2/3] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:13:52 2016] GET /static/css/style.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6491|app: 0|req: 1/4] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:13:52 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 12 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6490|app: 0|req: 1/5] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:13:52 2016] GET /static/css/form-elements.css => generated 0 bytes in 8 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6489|app: 0|req: 3/6] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:13:52 2016] GET /static/js/jquery.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6493|app: 0|req: 2/7] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:13:52 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 8 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6492|app: 0|req: 1/8] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:13:52 2016] GET /static/js/scripts.js => generated 0 bytes in 9 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6489|app: 0|req: 4/9] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:13:52 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6491|app: 0|req: 2/10] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:13:53 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 93, in register_process
    if user['status']:
TypeError: 'NoneType' object has no attribute '__getitem__'
[pid: 6490|app: 0|req: 2/11] 70.213.12.152 () {46 vars in 961 bytes} [Wed Jun 29 20:13:54 2016] POST /register/process => generated 0 bytes in 6 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:14:54 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x26cd060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x26cd060 pid: 6534 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6534)
spawned uWSGI worker 1 (pid: 6538, cores: 1)
spawned uWSGI worker 2 (pid: 6539, cores: 1)
spawned uWSGI worker 3 (pid: 6540, cores: 1)
spawned uWSGI worker 4 (pid: 6541, cores: 1)
spawned uWSGI worker 5 (pid: 6542, cores: 1)
2016-06-29 20:14:56,877 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'sql_mode'
2016-06-29 20:14:56,877 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:14:56,878 INFO sqlalchemy.engine.base.Engine SELECT DATABASE()
2016-06-29 20:14:56,878 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:14:56,878 INFO sqlalchemy.engine.base.Engine show collation where `Charset` = 'utf8' and `Collation` = 'utf8_bin'
2016-06-29 20:14:56,878 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:14:56,880 INFO sqlalchemy.engine.base.Engine SELECT CAST('test plain returns' AS CHAR(60)) AS anon_1
2016-06-29 20:14:56,880 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:14:56,880 INFO sqlalchemy.engine.base.Engine SELECT CAST('test unicode returns' AS CHAR(60)) AS anon_1
2016-06-29 20:14:56,880 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:14:56,881 INFO sqlalchemy.engine.base.Engine SELECT CAST('test collated returns' AS CHAR CHARACTER SET utf8) COLLATE utf8_bin AS anon_1
2016-06-29 20:14:56,881 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:14:56,881 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:14:56,882 INFO sqlalchemy.engine.base.Engine UPDATE users SET email = %s, username = %s WHERE fb_user_id = %s
2016-06-29 20:14:56,882 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com', u'chaosrock', u'10153941387519143')
2016-06-29 20:14:56,884 INFO sqlalchemy.engine.base.Engine COMMIT
2016-06-29 20:14:56,884 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:14:56,884 INFO sqlalchemy.engine.base.Engine SELECT * FROM users WHERE email = %s
2016-06-29 20:14:56,884 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com',)
2016-06-29 20:14:56,891 INFO sqlalchemy.engine.base.Engine ROLLBACK
[pid: 6542|app: 0|req: 1/1] 70.213.12.152 () {46 vars in 961 bytes} [Wed Jun 29 20:14:56 2016] POST /register/process => generated 209 bytes in 36 msecs (HTTP/1.1 302) 4 headers in 448 bytes (1 switches on core 0)
[pid: 6538|app: 0|req: 1/2] 70.213.12.152 () {42 vars in 953 bytes} [Wed Jun 29 20:14:56 2016] GET / => generated 3164 bytes in 23 msecs (HTTP/1.1 200) 3 headers in 412 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:19:04 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x25a9060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x25a9060 pid: 6590 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6590)
spawned uWSGI worker 1 (pid: 6594, cores: 1)
spawned uWSGI worker 2 (pid: 6595, cores: 1)
spawned uWSGI worker 3 (pid: 6596, cores: 1)
spawned uWSGI worker 4 (pid: 6597, cores: 1)
spawned uWSGI worker 5 (pid: 6598, cores: 1)
[pid: 6598|app: 0|req: 1/1] 70.213.12.152 () {44 vars in 984 bytes} [Wed Jun 29 20:19:06 2016] GET / => generated 3312 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6594|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:19:06 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6595|app: 0|req: 1/3] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:19:06 2016] GET /static/css/form-elements.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6594|app: 0|req: 2/4] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:19:06 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6594|app: 0|req: 3/5] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:19:06 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6598|app: 0|req: 2/6] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:19:06 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 8 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6596|app: 0|req: 1/7] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:19:06 2016] GET /static/js/jquery.min.js => generated 0 bytes in 14 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6597|app: 0|req: 1/8] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:19:06 2016] GET /static/css/style.css => generated 0 bytes in 18 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6594|app: 0|req: 4/9] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:19:06 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6595|app: 0|req: 2/10] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:19:06 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6598|app: 0|req: 3/11] 70.213.12.152 () {44 vars in 845 bytes} [Wed Jun 29 20:19:12 2016] GET / => generated 3183 bytes in 2 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6594|app: 0|req: 5/12] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:19:12 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6595|app: 0|req: 3/13] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:19:12 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6596|app: 0|req: 2/14] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:19:12 2016] GET /static/css/form-elements.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6597|app: 0|req: 2/15] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:19:12 2016] GET /static/css/style.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6598|app: 0|req: 4/16] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:19:12 2016] GET /static/js/jquery.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6594|app: 0|req: 6/17] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:19:12 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6595|app: 0|req: 4/18] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:19:12 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6596|app: 0|req: 3/19] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:19:12 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6598|app: 0|req: 5/20] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:19:12 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:19:35 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x12a1060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x12a1060 pid: 6638 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6638)
spawned uWSGI worker 1 (pid: 6642, cores: 1)
spawned uWSGI worker 2 (pid: 6643, cores: 1)
spawned uWSGI worker 3 (pid: 6644, cores: 1)
spawned uWSGI worker 4 (pid: 6645, cores: 1)
spawned uWSGI worker 5 (pid: 6646, cores: 1)
[pid: 6645|app: 0|req: 1/1] 70.213.12.152 () {44 vars in 845 bytes} [Wed Jun 29 20:19:36 2016] GET / => generated 3195 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6644|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:19:37 2016] GET /static/css/style.css => generated 0 bytes in 16 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6643|app: 0|req: 1/3] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:19:37 2016] GET /static/css/form-elements.css => generated 0 bytes in 23 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6646|app: 0|req: 1/4] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:19:37 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 26 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6642|app: 0|req: 1/5] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:19:37 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 31 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6645|app: 0|req: 2/6] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:19:37 2016] GET /static/js/jquery.min.js => generated 0 bytes in 8 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6644|app: 0|req: 2/7] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:19:37 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 2 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6643|app: 0|req: 2/8] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:19:37 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6646|app: 0|req: 2/9] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:19:37 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6642|app: 0|req: 2/10] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:19:37 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
2016-06-29 20:19:42,378 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'sql_mode'
2016-06-29 20:19:42,378 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:19:42,379 INFO sqlalchemy.engine.base.Engine SELECT DATABASE()
2016-06-29 20:19:42,379 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:19:42,379 INFO sqlalchemy.engine.base.Engine show collation where `Charset` = 'utf8' and `Collation` = 'utf8_bin'
2016-06-29 20:19:42,379 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:19:42,380 INFO sqlalchemy.engine.base.Engine SELECT CAST('test plain returns' AS CHAR(60)) AS anon_1
2016-06-29 20:19:42,380 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:19:42,381 INFO sqlalchemy.engine.base.Engine SELECT CAST('test unicode returns' AS CHAR(60)) AS anon_1
2016-06-29 20:19:42,381 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:19:42,381 INFO sqlalchemy.engine.base.Engine SELECT CAST('test collated returns' AS CHAR CHARACTER SET utf8) COLLATE utf8_bin AS anon_1
2016-06-29 20:19:42,382 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:19:42,382 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:19:42,382 INFO sqlalchemy.engine.base.Engine UPDATE users SET email = %s, username = %s WHERE fb_user_id = %s
2016-06-29 20:19:42,382 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com', u'chaosrock', u'10153941387519143')
2016-06-29 20:19:42,383 INFO sqlalchemy.engine.base.Engine COMMIT
2016-06-29 20:19:42,383 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:19:42,384 INFO sqlalchemy.engine.base.Engine SELECT * FROM users WHERE email = %s
2016-06-29 20:19:42,384 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com',)
2016-06-29 20:19:42,390 INFO sqlalchemy.engine.base.Engine ROLLBACK
[pid: 6645|app: 0|req: 3/11] 70.213.12.152 () {46 vars in 961 bytes} [Wed Jun 29 20:19:42 2016] POST /register/process => generated 209 bytes in 29 msecs (HTTP/1.1 302) 4 headers in 448 bytes (1 switches on core 0)
[pid: 6642|app: 0|req: 3/12] 70.213.12.152 () {42 vars in 953 bytes} [Wed Jun 29 20:19:42 2016] GET / => generated 3340 bytes in 22 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:20:54 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2077060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x2077060 pid: 6687 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6687)
spawned uWSGI worker 1 (pid: 6691, cores: 1)
spawned uWSGI worker 2 (pid: 6692, cores: 1)
spawned uWSGI worker 3 (pid: 6693, cores: 1)
spawned uWSGI worker 4 (pid: 6694, cores: 1)
spawned uWSGI worker 5 (pid: 6695, cores: 1)
[pid: 6695|app: 0|req: 1/1] 70.213.12.152 () {44 vars in 845 bytes} [Wed Jun 29 20:20:56 2016] GET / => generated 3198 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6691|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:20:56 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6695|app: 0|req: 2/3] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:20:56 2016] GET /static/css/style.css => generated 0 bytes in 15 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6691|app: 0|req: 2/4] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:20:56 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 11 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6695|app: 0|req: 3/5] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:20:56 2016] GET /static/js/scripts.js => generated 0 bytes in 3 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6694|app: 0|req: 1/6] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:20:56 2016] GET /static/css/form-elements.css => generated 0 bytes in 22 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6692|app: 0|req: 1/7] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:20:56 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 26 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6693|app: 0|req: 1/8] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:20:56 2016] GET /static/js/jquery.min.js => generated 0 bytes in 15 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6691|app: 0|req: 3/9] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:20:56 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6695|app: 0|req: 4/10] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:20:56 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
2016-06-29 20:21:05,249 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'sql_mode'
2016-06-29 20:21:05,249 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:21:05,250 INFO sqlalchemy.engine.base.Engine SELECT DATABASE()
2016-06-29 20:21:05,250 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:21:05,250 INFO sqlalchemy.engine.base.Engine show collation where `Charset` = 'utf8' and `Collation` = 'utf8_bin'
2016-06-29 20:21:05,250 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:21:05,253 INFO sqlalchemy.engine.base.Engine SELECT CAST('test plain returns' AS CHAR(60)) AS anon_1
2016-06-29 20:21:05,253 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:21:05,253 INFO sqlalchemy.engine.base.Engine SELECT CAST('test unicode returns' AS CHAR(60)) AS anon_1
2016-06-29 20:21:05,253 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:21:05,254 INFO sqlalchemy.engine.base.Engine SELECT CAST('test collated returns' AS CHAR CHARACTER SET utf8) COLLATE utf8_bin AS anon_1
2016-06-29 20:21:05,254 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:21:05,255 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:21:05,255 INFO sqlalchemy.engine.base.Engine UPDATE users SET email = %s, username = %s WHERE fb_user_id = %s
2016-06-29 20:21:05,255 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com', u'chaosrock', u'10153941387519143')
2016-06-29 20:21:05,255 INFO sqlalchemy.engine.base.Engine COMMIT
2016-06-29 20:21:05,256 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:21:05,256 INFO sqlalchemy.engine.base.Engine SELECT * FROM users WHERE email = %s
2016-06-29 20:21:05,256 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com',)
2016-06-29 20:21:05,262 INFO sqlalchemy.engine.base.Engine ROLLBACK
[pid: 6694|app: 0|req: 2/11] 70.213.12.152 () {46 vars in 961 bytes} [Wed Jun 29 20:21:05 2016] POST /register/process => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 448 bytes (1 switches on core 0)
[pid: 6691|app: 0|req: 4/12] 70.213.12.152 () {42 vars in 953 bytes} [Wed Jun 29 20:21:05 2016] GET / => generated 3343 bytes in 22 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6692|app: 0|req: 2/13] 70.213.12.152 () {44 vars in 845 bytes} [Wed Jun 29 20:21:33 2016] GET / => generated 3198 bytes in 22 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6691|app: 0|req: 5/14] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:21:33 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6693|app: 0|req: 2/15] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:21:33 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6695|app: 0|req: 5/16] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:21:33 2016] GET /static/css/form-elements.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6694|app: 0|req: 3/17] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:21:33 2016] GET /static/css/style.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6695|app: 0|req: 6/18] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:21:33 2016] GET /static/js/jquery.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6694|app: 0|req: 4/19] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:21:33 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6693|app: 0|req: 3/20] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:21:33 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6691|app: 0|req: 6/21] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:21:33 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6695|app: 0|req: 7/22] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:21:34 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:23:16 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1cf8060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1cf8060 pid: 6737 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6737)
spawned uWSGI worker 1 (pid: 6741, cores: 1)
spawned uWSGI worker 2 (pid: 6742, cores: 1)
spawned uWSGI worker 3 (pid: 6743, cores: 1)
spawned uWSGI worker 4 (pid: 6744, cores: 1)
spawned uWSGI worker 5 (pid: 6745, cores: 1)
[pid: 6745|app: 0|req: 1/1] 70.213.12.152 () {44 vars in 844 bytes} [Wed Jun 29 20:23:38 2016] GET / => generated 3127 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6741|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 971 bytes} [Wed Jun 29 20:23:38 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6745|app: 0|req: 2/3] 70.213.12.152 () {48 vars in 969 bytes} [Wed Jun 29 20:23:38 2016] GET /static/css/form-elements.css => generated 0 bytes in 8 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6742|app: 0|req: 1/4] 70.213.12.152 () {48 vars in 975 bytes} [Wed Jun 29 20:23:38 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 10 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6741|app: 0|req: 2/5] 70.213.12.152 () {48 vars in 953 bytes} [Wed Jun 29 20:23:38 2016] GET /static/css/style.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6744|app: 0|req: 1/6] 70.213.12.152 () {48 vars in 945 bytes} [Wed Jun 29 20:23:38 2016] GET /static/js/jquery.min.js => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6741|app: 0|req: 3/7] 70.213.12.152 () {48 vars in 950 bytes} [Wed Jun 29 20:23:38 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6742|app: 0|req: 2/8] 70.213.12.152 () {48 vars in 935 bytes} [Wed Jun 29 20:23:38 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6745|app: 0|req: 3/9] 70.213.12.152 () {48 vars in 973 bytes} [Wed Jun 29 20:23:39 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6743|app: 0|req: 1/10] 70.213.12.152 () {48 vars in 971 bytes} [Wed Jun 29 20:23:39 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 8 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
2016-06-29 20:23:43,839 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'sql_mode'
2016-06-29 20:23:43,839 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:23:43,840 INFO sqlalchemy.engine.base.Engine SELECT DATABASE()
2016-06-29 20:23:43,840 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:23:43,840 INFO sqlalchemy.engine.base.Engine show collation where `Charset` = 'utf8' and `Collation` = 'utf8_bin'
2016-06-29 20:23:43,840 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:23:43,841 INFO sqlalchemy.engine.base.Engine SELECT CAST('test plain returns' AS CHAR(60)) AS anon_1
2016-06-29 20:23:43,841 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:23:43,842 INFO sqlalchemy.engine.base.Engine SELECT CAST('test unicode returns' AS CHAR(60)) AS anon_1
2016-06-29 20:23:43,842 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:23:43,842 INFO sqlalchemy.engine.base.Engine SELECT CAST('test collated returns' AS CHAR CHARACTER SET utf8) COLLATE utf8_bin AS anon_1
2016-06-29 20:23:43,842 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:23:43,843 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:23:43,843 INFO sqlalchemy.engine.base.Engine UPDATE users SET email = %s, username = %s WHERE fb_user_id = %s
2016-06-29 20:23:43,843 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com', u'chaosrock', u'10153941387519143')
2016-06-29 20:23:43,844 INFO sqlalchemy.engine.base.Engine COMMIT
2016-06-29 20:23:43,844 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:23:43,844 INFO sqlalchemy.engine.base.Engine SELECT * FROM users WHERE email = %s
2016-06-29 20:23:43,844 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com',)
2016-06-29 20:23:43,851 INFO sqlalchemy.engine.base.Engine ROLLBACK
[pid: 6741|app: 0|req: 4/11] 70.213.12.152 () {46 vars in 960 bytes} [Wed Jun 29 20:23:43 2016] POST /register/process => generated 209 bytes in 35 msecs (HTTP/1.1 302) 4 headers in 448 bytes (1 switches on core 0)
[pid: 6742|app: 0|req: 3/12] 70.213.12.152 () {42 vars in 952 bytes} [Wed Jun 29 20:23:43 2016] GET / => generated 3272 bytes in 23 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:25:21 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1d9e060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1d9e060 pid: 6799 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6799)
spawned uWSGI worker 1 (pid: 6803, cores: 1)
spawned uWSGI worker 2 (pid: 6804, cores: 1)
spawned uWSGI worker 3 (pid: 6805, cores: 1)
spawned uWSGI worker 4 (pid: 6806, cores: 1)
spawned uWSGI worker 5 (pid: 6807, cores: 1)
[pid: 6807|app: 0|req: 1/1] 70.213.12.152 () {44 vars in 845 bytes} [Wed Jun 29 20:25:23 2016] GET / => generated 3127 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 273 bytes (1 switches on core 0)
[pid: 6803|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:25:23 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6806|app: 0|req: 1/3] 70.213.12.152 () {48 vars in 954 bytes} [Wed Jun 29 20:25:23 2016] GET /static/css/style.css => generated 0 bytes in 9 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6804|app: 0|req: 1/4] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:25:23 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 12 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6803|app: 0|req: 2/5] 70.213.12.152 () {48 vars in 970 bytes} [Wed Jun 29 20:25:23 2016] GET /static/css/form-elements.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6807|app: 0|req: 2/6] 70.213.12.152 () {48 vars in 946 bytes} [Wed Jun 29 20:25:23 2016] GET /static/js/jquery.min.js => generated 0 bytes in 4 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
[pid: 6805|app: 0|req: 1/7] 70.213.12.152 () {48 vars in 951 bytes} [Wed Jun 29 20:25:23 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6803|app: 0|req: 3/8] 70.213.12.152 () {48 vars in 936 bytes} [Wed Jun 29 20:25:23 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 378 bytes (0 switches on core 0)
[pid: 6804|app: 0|req: 2/9] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:25:23 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 381 bytes (0 switches on core 0)
[pid: 6807|app: 0|req: 3/10] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:25:23 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 382 bytes (0 switches on core 0)
2016-06-29 20:25:28,394 INFO sqlalchemy.engine.base.Engine SHOW VARIABLES LIKE 'sql_mode'
2016-06-29 20:25:28,394 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:25:28,395 INFO sqlalchemy.engine.base.Engine SELECT DATABASE()
2016-06-29 20:25:28,395 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:25:28,395 INFO sqlalchemy.engine.base.Engine show collation where `Charset` = 'utf8' and `Collation` = 'utf8_bin'
2016-06-29 20:25:28,395 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:25:28,398 INFO sqlalchemy.engine.base.Engine SELECT CAST('test plain returns' AS CHAR(60)) AS anon_1
2016-06-29 20:25:28,398 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:25:28,398 INFO sqlalchemy.engine.base.Engine SELECT CAST('test unicode returns' AS CHAR(60)) AS anon_1
2016-06-29 20:25:28,398 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:25:28,399 INFO sqlalchemy.engine.base.Engine SELECT CAST('test collated returns' AS CHAR CHARACTER SET utf8) COLLATE utf8_bin AS anon_1
2016-06-29 20:25:28,399 INFO sqlalchemy.engine.base.Engine ()
2016-06-29 20:25:28,400 INFO sqlalchemy.engine.base.Engine BEGIN (implicit)
2016-06-29 20:25:28,400 INFO sqlalchemy.engine.base.Engine UPDATE users SET email = %s, username = %s WHERE fb_user_id = %s
2016-06-29 20:25:28,400 INFO sqlalchemy.engine.base.Engine (u'josiahrooney@gmail.com', u'chaosrock', u'10153941387519143')
2016-06-29 20:25:28,400 INFO sqlalchemy.engine.base.Engine COMMIT
[pid: 6806|app: 0|req: 2/11] 70.213.12.152 () {46 vars in 961 bytes} [Wed Jun 29 20:25:28 2016] POST /register/process => generated 209 bytes in 31 msecs (HTTP/1.1 302) 4 headers in 311 bytes (1 switches on core 0)
[pid: 6803|app: 0|req: 4/12] 70.213.12.152 () {42 vars in 816 bytes} [Wed Jun 29 20:25:28 2016] GET / => generated 3127 bytes in 24 msecs (HTTP/1.1 200) 3 headers in 275 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket ridr.sock removed.
*** Starting uWSGI 2.0.13.1 (64bit) on [Wed Jun 29 20:32:30 2016] ***
compiled with version: 4.8.4 on 23 June 2016 23:01:39
os: Linux-3.13.0-74-generic #118-Ubuntu SMP Thu Dec 17 22:52:10 UTC 2015
nodename: ip-172-31-2-234
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ubuntu/ridr
detected binary path: /home/ubuntu/ridr/pylotVenv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your processes number limit is 7862
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ridr.sock fd 3
Python version: 2.7.6 (default, Jun 22 2015, 18:01:27)  [GCC 4.8.2]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xcf8060
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 436560 bytes (426 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xcf8060 pid: 6858 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 6858)
spawned uWSGI worker 1 (pid: 6862, cores: 1)
spawned uWSGI worker 2 (pid: 6863, cores: 1)
spawned uWSGI worker 3 (pid: 6864, cores: 1)
spawned uWSGI worker 4 (pid: 6865, cores: 1)
spawned uWSGI worker 5 (pid: 6866, cores: 1)
[pid: 6864|app: 0|req: 1/1] 70.213.12.152 () {44 vars in 847 bytes} [Wed Jun 29 20:32:36 2016] GET / => generated 3153 bytes in 25 msecs (HTTP/1.1 200) 3 headers in 275 bytes (1 switches on core 0)
[pid: 6862|app: 0|req: 1/2] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:32:36 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 385 bytes (0 switches on core 0)
[pid: 6863|app: 0|req: 1/3] 70.213.12.152 () {48 vars in 978 bytes} [Wed Jun 29 20:32:36 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6862|app: 0|req: 2/4] 70.213.12.152 () {48 vars in 972 bytes} [Wed Jun 29 20:32:37 2016] GET /static/css/form-elements.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6866|app: 0|req: 1/5] 70.213.12.152 () {48 vars in 956 bytes} [Wed Jun 29 20:32:37 2016] GET /static/css/style.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6862|app: 0|req: 3/6] 70.213.12.152 () {48 vars in 947 bytes} [Wed Jun 29 20:32:37 2016] GET /static/js/jquery.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 384 bytes (0 switches on core 0)
[pid: 6863|app: 0|req: 2/7] 70.213.12.152 () {48 vars in 953 bytes} [Wed Jun 29 20:32:37 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6864|app: 0|req: 2/8] 70.213.12.152 () {48 vars in 937 bytes} [Wed Jun 29 20:32:37 2016] GET /static/js/scripts.js => generated 0 bytes in 4 msecs (HTTP/1.1 304) 5 headers in 380 bytes (0 switches on core 0)
[pid: 6865|app: 0|req: 1/9] 70.213.12.152 () {48 vars in 976 bytes} [Wed Jun 29 20:32:37 2016] GET /static/css/fonts.css => generated 0 bytes in 6 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6862|app: 0|req: 4/10] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:32:37 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 384 bytes (0 switches on core 0)
[pid: 6863|app: 0|req: 3/11] 70.213.12.152 () {44 vars in 847 bytes} [Wed Jun 29 20:32:44 2016] GET / => generated 3153 bytes in 23 msecs (HTTP/1.1 200) 3 headers in 275 bytes (1 switches on core 0)
[pid: 6862|app: 0|req: 5/12] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:32:45 2016] GET /static/css/bootstrap.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 385 bytes (0 switches on core 0)
[pid: 6866|app: 0|req: 2/13] 70.213.12.152 () {48 vars in 978 bytes} [Wed Jun 29 20:32:45 2016] GET /static/css/font-awesome.min.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6865|app: 0|req: 2/14] 70.213.12.152 () {48 vars in 971 bytes} [Wed Jun 29 20:32:45 2016] GET /static/css/form-elements.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6864|app: 0|req: 3/15] 70.213.12.152 () {48 vars in 955 bytes} [Wed Jun 29 20:32:45 2016] GET /static/css/style.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6864|app: 0|req: 4/16] 70.213.12.152 () {48 vars in 948 bytes} [Wed Jun 29 20:32:45 2016] GET /static/js/jquery.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 384 bytes (0 switches on core 0)
[pid: 6865|app: 0|req: 3/17] 70.213.12.152 () {48 vars in 953 bytes} [Wed Jun 29 20:32:45 2016] GET /static/js/bootstrap.min.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6866|app: 0|req: 3/18] 70.213.12.152 () {48 vars in 938 bytes} [Wed Jun 29 20:32:45 2016] GET /static/js/scripts.js => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 380 bytes (0 switches on core 0)
[pid: 6862|app: 0|req: 6/19] 70.213.12.152 () {48 vars in 975 bytes} [Wed Jun 29 20:32:45 2016] GET /static/css/fonts.css => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 383 bytes (0 switches on core 0)
[pid: 6864|app: 0|req: 5/20] 70.213.12.152 () {48 vars in 974 bytes} [Wed Jun 29 20:32:45 2016] GET /static/images/ridr_transparent-1.png => generated 0 bytes in 1 msecs (HTTP/1.1 304) 5 headers in 384 bytes (0 switches on core 0)
[pid: 6863|app: 0|req: 4/21] 70.213.12.152 () {42 vars in 828 bytes} [Wed Jun 29 20:32:47 2016] GET /logout => generated 209 bytes in 6 msecs (HTTP/1.1 302) 4 headers in 197 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 7/22] 70.213.12.152 () {40 vars in 639 bytes} [Wed Jun 29 20:32:47 2016] GET / => generated 0 bytes in 25 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6863|app: 0|req: 5/23] 70.213.12.152 () {42 vars in 670 bytes} [Wed Jun 29 20:32:50 2016] GET / => generated 0 bytes in 4 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 8/24] 70.213.12.152 () {40 vars in 692 bytes} [Wed Jun 29 21:07:44 2016] GET / => generated 0 bytes in 3 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 9/25] 12.199.203.211 () {42 vars in 706 bytes} [Wed Jun 29 22:46:08 2016] GET / => generated 0 bytes in 4 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6863|app: 0|req: 6/26] 12.199.203.211 () {46 vars in 759 bytes} [Wed Jun 29 22:46:15 2016] GET / => generated 0 bytes in 3 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 6866|app: 0|req: 4/27] 12.199.203.211 () {46 vars in 713 bytes} [Wed Jun 29 22:46:15 2016] GET /favicon.ico => generated 233 bytes in 1 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 10/28] 110.88.44.150 () {36 vars in 560 bytes} [Wed Jun 29 23:32:25 2016] GET / => generated 0 bytes in 4 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 11/29] 169.54.244.75 () {30 vars in 393 bytes} [Thu Jun 30 00:59:40 2016] GET / => generated 0 bytes in 3 msecs (HTTP/1.0 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 12/30] 158.69.229.134 () {32 vars in 438 bytes} [Thu Jun 30 01:13:45 2016] GET / => generated 0 bytes in 3 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6863|app: 0|req: 7/31] 12.199.203.211 () {42 vars in 706 bytes} [Thu Jun 30 01:13:59 2016] GET / => generated 0 bytes in 3 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 6866|app: 0|req: 5/32] 12.199.203.211 () {42 vars in 660 bytes} [Thu Jun 30 01:13:59 2016] GET /favicon.ico => generated 233 bytes in 0 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 13/33] 107.77.109.34 () {42 vars in 769 bytes} [Thu Jun 30 02:58:30 2016] GET / => generated 0 bytes in 4 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 6863|app: 0|req: 8/34] 107.77.109.34 () {42 vars in 723 bytes} [Thu Jun 30 02:58:30 2016] GET /favicon.ico => generated 233 bytes in 1 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6866|app: 0|req: 6/35] 66.220.145.243 () {44 vars in 825 bytes} [Thu Jun 30 02:58:33 2016] GET / => generated 0 bytes in 25 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 6862|app: 0|req: 14/36] 66.220.145.243 () {42 vars in 634 bytes} [Thu Jun 30 02:58:34 2016] GET /favicon.ico => generated 233 bytes in 1 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6863|app: 0|req: 9/37] 73.66.10.61 () {40 vars in 714 bytes} [Thu Jun 30 02:58:40 2016] GET / => generated 0 bytes in 3 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 6864|app: 0|req: 6/38] 73.66.10.61 () {40 vars in 668 bytes} [Thu Jun 30 02:58:42 2016] GET /favicon.ico => generated 233 bytes in 1 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6865|app: 0|req: 4/39] 73.66.10.61 () {40 vars in 714 bytes} [Thu Jun 30 02:59:03 2016] GET / => generated 0 bytes in 25 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6862|app: 0|req: 15/40] 73.66.10.61 () {42 vars in 745 bytes} [Thu Jun 30 02:59:05 2016] GET / => generated 0 bytes in 4 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "./system/core/controller.py", line 36, in dispatch_request
    return action(*args, **kwargs)
  File "./app/controllers/Users.py", line 36, in index
    return self.load_view('index.html')
  File "./system/core/controller.py", line 43, in load_view
    return render_template(view_name, **kwargs)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/ridr/app/views/index.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/home/ubuntu/ridr/app/views/layout.html", line 62, in top-level template code
    {% block content %}{% endblock %}
  File "/home/ubuntu/ridr/app/views/index.html", line 6, in block "content"
    {{ session['user']['id'] }}
  File "/home/ubuntu/ridr/pylotVenv/local/lib/python2.7/site-packages/jinja2/environment.py", line 389, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'flask.sessions.SecureCookieSession object' has no attribute 'user'
[pid: 6863|app: 0|req: 10/41] 73.66.10.61 () {44 vars in 762 bytes} [Thu Jun 30 02:59:45 2016] GET / => generated 0 bytes in 4 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
